content/news/2024-09-30-maplibre-newsletter-september-2024/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "MapLibre Newsletter September 2024"
+date: "2024-09-30"
+categories: ["newsletter"]
+authors: [ramyaragupathy, ianthetechie, harel, bart]
+draft: false
+---
+
+# MapLibre Newsletter - September 2024
+
+## SwiftUI DSL
```

Comment:
Lets move this section to after MapLibre GL JS and MapLibre Nstive since these are the core projects

Reply:
moved